docs/continuous-integration.md

```diff
@@ -0,0 +1,45 @@
+
+# Continuous integration
+
+We use [Konflux][1] for CI, and own the 'ocm-ui-tenant' namespace.  This namespace hosts the ['ocm-ui' application][8], which in turn holds the ['uhc-portal' build component][12] pointing at this code repo.
+            
+
+## Build pipelines
+
+The FE app is built onto container images, which are pushed into the quay.io registry; PR changes are built via the ['pull-request' pipeline][3] and long-lived branches (e.g. 'main') – via the ['push' pipeline][4] (see ['pipeline runs'][9] in Konflux UI).  Both are stored in an [image repo under the redhat-user-workloads org][2].
+
+Konflux will take care to take [snapshots][5] that can be used for testing, and to create [releases][6] which will be stored in an [image repo under the redhat-services-prod org][7].  Konflux releases are just images that have been signed as suitable for deployment.
+        
+
+## Persisted configuration
+
+Most of our Konflux configuration (e.g. our app, component, integration-test scenario, release plans, user access) is persisted as yaml files in the [konflux-release-data][10] repository, and then parsed and displayed by Konflux UI.  
+See there for more info on how to generate or update these configs.
+
+
+## Our custom setup
+
+Our repo declares a non-default pipeline-run config, which extends a remote pipeline config commonly used in HCC tenant-apps.  This config is equipped with an additional custom task for running source-code verification during build (e.g. unit-tests, linter).
+
+See [docker-build-run-unit-tests][11] at the _RedHatInsights/konflux-pipelines_ repo.
+
+ 
+## Getting help
+
+To get assistance with Konflux CI, post an ask in the #konflux-users Slack channel.
```

Comment:
Do we have any links to hcc docs about konflux or konflux docs in general that you found useful and that you would add here too?

Reply:
there are links to konflux-docs at the top.  i don't know of any HCC docs outside of the migration-guide (which is awful TBH – it was often out-of-date or misleading).

i didn't have any specific pages to put under 'getting help'..  maybe the 'FAQ' page?  yes i'll add that ✏️